dtfs-central-api/src/services/state-machines/utilisation-report/event-handlers/helpers/calculate-initial-utilisation-and-fixed-fee.test.ts

```diff
@@ -1,10 +1,13 @@
 import * as dtfsCommon from '@ukef/dtfs2-common';
 import { calculateInitialUtilisationAndFixedFee, parseDate, hasRequiredValues, RequiredParams } from './calculate-initial-utilisation-and-fixed-fee';
-import { TfmFacilitiesRepo } from '../../../../../repositories/tfm-facilities-repo';
+import { NotFoundError } from '../../../../../errors';
 import { aTfmFacility } from '../../../../../../test-helpers';
 import * as fixedFeeHelpers from './calculate-initial-fixed-fee';
+import * as helpers from '../../../../../helpers';
 
 jest.mock('./calculate-initial-fixed-fee');
+jest.mock('../../../../../helpers/get-keying-sheet-calculation-tfm-facility-values');
```

Comment:
Why do we need to do this please, looks like the main helper is not even importing this

Reply:
removed